Formula/cnosdb.rb

```diff
@@ -0,0 +1,82 @@
+class Cnosdb < Formula
+  desc "Open source distributed time series database"
+  homepage "https://www.cnosdb.com"
+  url "https://github.com/cnosdb/cnosdb/archive/refs/tags/v1.0.2.tar.gz"
+  sha256 "ae412d0944b64c9b39dc1edc66f7b6f712b85bc5afad354c12b135ae71017100"
+  license "MIT"
+  head "https://github.com/cnosdb/cnosdb.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOBIN"] = buildpath
+    system "go", "install", "./..."
+    bin.install %w[cnosdb cnosdb-cli cnosdb-ctl cnosdb-meta cnosdb-inspect cnosdb-tools]
+    etc.install "etc/cnosdb.sample.toml" => "cnosdb.conf"
+    inreplace etc/"cnosdb.conf" do |s|
+      s.gsub! "/var/lib/cnosdb/data", "#{var}/cnosdb/data"
+      s.gsub! "/var/lib/cnosdb/meta", "#{var}/cnosdb/meta"
+      s.gsub! "/var/lib/cnosdb/wal", "#{var}/cnosdb/wal"
+    end
+    (var/"cnosdb/data").mkpath
+    (var/"cnosdb/meta").mkpath
+    (var/"cnosdb/wal").mkpath
+  end
+  plist_options :manual => "cnosdb --config #{HOMEBREW_PREFIX}/etc/cnosdb.conf"
+  def plist
```

Comment:
This should be a service block, not plist. Those don't work on Linux.

Reply:
Thank you bro. I've had this puzzle for a long time.